lib/pages/login/login_page.dart

```diff
@@ -23,54 +25,62 @@ class LoginPage extends StatelessWidget {
     double smallGap = height * 0.01;
     double messageFieldWidth = 0.85;
 
-    return Scaffold(
-      body: SizedBox(
-        width: width,
-        child: Column(
-          mainAxisAlignment: MainAxisAlignment.center,
-          crossAxisAlignment: CrossAxisAlignment.center,
-          children: [
-            CircleAvatar(
-              backgroundColor: Theme.of(context).scaffoldBackgroundColor,
-              radius: avatarRadius,
-              child: Image.asset(Assets.efficacyAdminLogoImagePath),
-            ),
-            Text(
-              "Hey! Welcome",
-              style: Theme.of(context).textTheme.displaySmall,
-            ),
-            FractionallySizedBox(
-              widthFactor: messageFieldWidth,
-              child: Text(
-                "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suscipit sed augue quam amet, sed gravida.",
-                textAlign: TextAlign.center,
-                style: Theme.of(context).textTheme.bodyLarge,
+    return WillPopScope(
```

Comment:
WillPopScope is depricated try using an alternative of the suggested one of PopScope